src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -269,21 +271,21 @@ private BlockPos getWorkingPosition()
      */
     private BlockPos getWorkingPosition(int offset)
     {
-        if(offset > MAX_ADDITIONAL_RANGE_TO_BUILD)
+        if (offset > MAX_ADDITIONAL_RANGE_TO_BUILD)
         {
             return job.getSchematic().getBlockPosition();
         }
         //get length or width either is larger.
-        int length = job.getSchematic().getLength();
-        int width = job.getSchematic().getWidth();
-        int distance = width > length ? width : length;
+        int          length     = job.getSchematic().getLength();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'length' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)